app/controllers/actors/ProvisionerProcessor.scala

```diff
@@ -23,29 +22,33 @@ object ProvisionerStatus {
 
 case class ProvisionerResult(status: ProvisionerStatus, commandResult: CommandResult)
 
-case class ProvisionerTest(request: ProvisionerRequest, userTimeout: Option[FiniteDuration] = None)(implicit req: Request[AnyContent]) extends BackgroundProcess[ProvisionerResult]
+case class ProvisionerTest(request: ProvisionerRequest, userTimeout: Option[FiniteDuration] = None)(implicit req: Request[AnyContent], ec : ExecutionContext) extends BackgroundProcess[ProvisionerResult]
```

Comment:
ec is implicit here.  If you want to specify implicit parameters in a scala function, the parameter list needs to begin with keyword 'implicit', and then every parameter in that list is implicit.

http://stackoverflow.com/questions/10635406/defining-a-function-with-multiple-implicit-arguments-in-scala